src/thermo/Elements.cpp

```diff
@@ -280,25 +353,23 @@ string getElementSymbol(int atomicNumber)
 
 string getElementName(const std::string& ename)
 {
-    int numElements = numElementsDefined();
-    int numIsotopes = numIsotopesDefined();
     string symbol = trimCopy(ename);
-    for (int i = 0; i < numElements; i++) {
-        if (symbol == atomicWeightTable[i].symbol) {
-            return atomicWeightTable[i].fullName;
+    for (const auto& atom : atomicWeightTable) {
+        if (symbol == atom.symbol) {
+            return atom.fullName;
         }
```

Comment:
If the suggested `elementNames` map is adopted, this would simplify (same for a potential `isotopeNames` map below).